### PR TITLE
fix(integrations): Send oversized messages for SQS

### PR DESCRIPTION
**Pull request by dekinderfiets**
Fixes bug (wrong placed condition) that prevents oversized messages from being sent to SQS queue.

### CODE REVIEW

File: src/sentry_plugins/amazon_sqs/plugin.py [hunk 1]
```diff
@@ -183,7 +183,7 @@ def sqs_send_message(message):
 
             message = json.dumps(payload)
 
-            if len(message) > 256 * 1024:
+            if len(message) > 256 * 1024 and not s3_bucket:
```
Comment (scefali) [inline src/sentry_plugins/amazon_sqs/plugin.py:186]: @dekinderfiets I don't think this added condition is going to do anything. If `s3_bucket` is truthy, then our message is just `{"s3Url": url, "eventID": event.event_id}` in a JSON format. That's not going to be larger than the maximum event side of 256KB. And regardless, we'd see an error trying to add anything larger to the queue than 256KB so logging and quitting early is fine.

Comment (dekinderfiets) [reply src/sentry_plugins/amazon_sqs/plugin.py:186]: The whole purpose of the S3 bucket is storing events greater than 256KB, the thing is that if s3_bucket is truthy and the message is greater than 256KB, this if statement will be also truthy so the message will not be sent to SQS.

Comment (scefali) [reply src/sentry_plugins/amazon_sqs/plugin.py:186]: @dekinderfiets I think you misunderstand how this code works. When we have the `s3_bucket` option, we upload the data here: https://github.com/getsentry/sentry/blob/master/src/sentry_plugins/amazon_sqs/plugin.py#L178

After we upload the data to S3, which happens _before_  the check on the payload size, we assemble a new payload here: https://github.com/getsentry/sentry/blob/master/src/sentry_plugins/amazon_sqs/plugin.py#L182

Then we upload that small amount of data to SQS which has a reference to the object in S3 and the event id.

Comment (dekinderfiets) [reply src/sentry_plugins/amazon_sqs/plugin.py:186]: I agree, didn't noticed the new size of the payload. closing PR.

**Pull request closed without merging**